<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VesselEnhancement</class>
 <widget class="qSlicerWidget" name="VesselEnhancement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="VesselEnhancementCollapsibleGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>A) Vessel enhacement</string>
     </property>
     <property name="collapsedHeight">
      <number>5</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="2">
       <widget class="QToolButton" name="VesselEnhancementOutputSaveToolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Save to file the output image</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="VesselEnhancementInputNode1ComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the fixed image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Fixed</string>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="VesselEnhancementInput1Label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Input image volume 1</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="VesselEnhancementInputNode3ComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the fixed image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Fixed</string>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>NotVisible</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="VesselEnhancementInput2Label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Input image volume 2</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>NotVisible</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="VesselEnhancementInput3Label">
        <property name="text">
         <string>Input image volume 3</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>NotVisible</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="VesselEnhancementInputMaskLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Mask volume</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="VesselEnhancementMaskNodeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the fixed image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Fixed</string>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="VesselEnhancementObjectIDLabel">
        <property name="text">
         <string>Organ label</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="VesselEnhancementInputNode2ComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the fixed image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Fixed</string>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>NotVisible</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="qMRMLNodeComboBox" name="VesselEnhancementOutputNodeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the output image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showHidden">
         <bool>true</bool>
        </property>
        <property name="baseName">
         <string/>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="VesselEnhancementOutputLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Choose the &lt;/span&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Output skin model&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; here.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Enhanced volume</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
          <string></string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="VesselEnhancementGoToModulePushButton">
          <property name="toolTip">
           <string>Open the &quot;BrainsFit&quot; module</string>
          </property>
          <property name="text">
           <string>Go to Vessel Enhancement module</string>
          </property>
          <property name="workflow" stdset="0">
           <stringlist>
            <string>2</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="VesselEnhancementApplyPushButton">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Register image together&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Enhance</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="VesselEnhancementSaveToolButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Save the output image</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="workflow" stdset="0">
           <stringlist>
            <string>0</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="qMRMLLabelComboBox" name="VesselEnhancementObjectIDLabelComboBox">
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLLabelComboBox</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLLabelComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>VesselEnhancement</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VesselEnhancementInputNode1ComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VesselEnhancement</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VesselEnhancementOutputNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VesselEnhancementOutputSaveToolButton</sender>
   <signal>clicked()</signal>
   <receiver>VesselEnhancementSaveToolButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VesselEnhancement</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VesselEnhancementInputNode2ComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VesselEnhancement</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VesselEnhancementMaskNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VesselEnhancement</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VesselEnhancementInputNode3ComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
